app/scripts/background.js

```diff
@@ -1591,6 +1576,11 @@ export function setupController(
 
     controller.rejectAllPendingApprovals();
   }
+
+  updateNetworkControllerRpcFailoverFromRemoteFeatureFlagController(
```

Comment:
Hmm. This seems a bit out-of-place here. Really we could run this check for whether RPC Failover is enabled at any point after the controller is setup I think? Seems like a better fit for the controller init module? That would avoid this pattern of passing the controller as a parameter as well, which we've been trying to avoid.

Reply:
Yeah, this was before I asked you about the anti-anti-pattern. I'll change it to that.